beetsplug/web/__init__.py

```diff
@@ -218,6 +221,14 @@ def item_query(queries):
     return g.lib.items(queries)
 
 
+@app.route('/item/at_path/<path:path>')
+def item_at_path(path):
+    try:
+        return flask.jsonify(_rep(beets.library.Item.from_path('/' + path)))
```

Comment:
This looks like it will give the client access to any file on the host's filesystem—not just things in the beets database. Maybe it would make sense to construct a `PathQuery`, as in the `mpdstats` plugin, to look up an item by its path.